src/bin/carbonadod.rs

```diff
@@ -11,11 +11,18 @@ use log::error;
 enum Commands {
     /// Start storage provider node with configured frontends
     Start,
+    /// Derive the parity npub from nsec
+    ParityNpub {
+        /// Nostr secret key
+        #[arg(long)]
+        nsec: String,
+    },
```

Comment:
can 
#[arg(long)]
also be
#[arg(long = "nsec")]

Reply:
Hm, I don't think it needs to be, since that's kind of redundant, though it's neat that we can do that